bigbluebutton-html5/imports/api/audio/client/bridge/sip.js

```diff
@@ -1,85 +1,336 @@
-import { makeCall } from '/imports/ui/services/api';
-
+import VoiceUsers from '/imports/api/voice-users';
+import { Tracker } from 'meteor/tracker';
 import BaseAudioBridge from './base';
 
-const APP_CONFIG = Meteor.settings.public.app;
-const MEDIA_CONFIG = Meteor.settings.public.media;
+const MEDIA = Meteor.settings.public.media;
+const STUN_TURN_FETCH_URL = MEDIA.stunTurnServersFetchAddress;
+const MEDIA_TAG = MEDIA.mediaTag;
+const CALL_TRANSFER_TIMEOUT = MEDIA.callTransferTimeout;
+
+const fetchStunTurnServers = sessionToken =>
+  new Promise(async (resolve, reject) => {
```

Comment:
While I definitely think I should revisit this function, The reason I don't need the outer `Promise` is that I already have a `thenable` inside it and not necessarily because I'm using `async`. We might sometime stumble upon a case where we need the approach used here, though this is not the case